sherpa-onnx/c-api/c-api.cc

```diff
@@ -73,6 +73,14 @@ SherpaOnnxOnlineRecognizer *SherpaOnnxCreateOnlineRecognizer(
 
   recognizer_config.model_config.tokens =
       SHERPA_ONNX_OR(config->model_config.tokens, "");
+  if (config->model_config.tokens_buf &&
+      config->model_config.tokens_buf_size > 0) {
+    recognizer_config.model_config.tokens_buf = std::string(
+        config->model_config.tokens_buf, config->model_config.tokens_buf_size);
+  } else {
+    recognizer_config.model_config.tokens_buf = "";
+  }
```

Comment:
```suggestion
  }
```

`std::string` is default initialized so there is no need to re-initialize it to an empty string.